### DISCUSSION TITLE
Metadata for tabular file

### DISCUSSION THREAD

**Discussion by ViragMistry** [sdcTools/tauargus#4] [Q&A]
I wanted to create the 2 dimension table as under, 
![image](https://user-images.githubusercontent.com/49681026/206105434-53f33ca3-64ed-4ffd-b859-84025fc66d54.png)

Please guide me on table and metadata file structure. 

I tried couple of options and getting below mention output,
![image](https://user-images.githubusercontent.com/49681026/206166177-3dcd7547-aafd-4153-b01a-f95bc40b0d27.png)

**Comment by ppdewolf** [comment]
Depends a lot on the information you have available. First of all: do you have microdata or only tabular data for input?
Assuming microdata input: see section 5.1.1 and 5.1.2 of the manual for meta data for microdata input.
Assuming tabular data input: see section 5.1.4 of the manual for meta data for tabular input.

Using only the information you provide with your screenshots, assuming that the bold number is unsafe, you could use e.g.

`table.rda`:
```
<SEPARATOR> ";"
<SAFE> S
<UNSAFE> U
"Marital Status"
	<RECODABLE>
	<TOTCODE> "Total"
"Sex"
	<RECODABLE>
	<TOTCODE> "Total"
Respons
	<NUMERIC>
status
	<STATUS>
```
or alternatively, if it is a frequency table:
```
<SEPARATOR> ";"
<SAFE> S
<UNSAFE> U
"Marital Status"
	<RECODABLE>
	<TOTCODE> "Total"
"Sex"
	<RECODABLE>
	<TOTCODE> "Total"
freq
	<FREQUENCY>
status
	<STATUS>
```
and tabular data input `table.tab`:
```
"Total";"Total";70;S
"Total";"Male";48;S
"Total";"Female";22;S
"Married";"Total";55;S
"Married";"Male";38;S
"Married";"Female";17;S
"Divorced";"Total";11;S
"Divorced";"Male";7;S
"Divorced";"Female";4;S
"Single";"Total";4;S
"Single";"Male";3;S
"Single";"Female";1;U
```
Using the second `.rda` and the `.tab` as input for tau-argus, with "Use safety rules", "Maximum frequency", "Frequency=2" I get:
![afbeelding](https://user-images.githubusercontent.com/19729987/206441047-2eef8aa9-9411-44cb-bf98-1039b09b0df6.png)
